packages/broker/test/integration/plugins/operator/InspectRandomNodeHelper.test.ts

```diff
@@ -0,0 +1,167 @@
+            await (await operatorContract.stake(sponsorship.address, parseEther("100"))).wait()
+            logger.debug(`staked on sponsorship ${sponsorship.address}`)
+            await (await operatorContract.stake(sponsorship2.address, parseEther("100"))).wait()
+            logger.debug(`staked on sponsorship ${sponsorship2.address}`)
+
+            await waitForCondition(async (): Promise<boolean> => {
+                const res = await inspectRandomNodeHelper.getSponsorshipsOfOperator(toEthereumAddress(operatorContract.address), 0)
+                return res.length === 2
+            }, 10000, 1000)
+
+            await waitForCondition(async (): Promise<boolean> => {
+                const res = await inspectRandomNodeHelper.getOperatorsInSponsorship(toEthereumAddress(sponsorship.address), 0)
+                return res.length === 1
+            }, 10000, 1000)
+
+        })
+
+        it("works to flag through the inspectRandomNodeHelper", async () => {
+            const flagger = await createWalletAndDeployOperator(provider, config, theGraphUrl)
+            logger.trace('deployed flagger contract ' + flagger.operatorConfig.operatorContractAddress)
+            const target = await createWalletAndDeployOperator(provider, config, theGraphUrl)
+            logger.trace('deployed target contract ' + target.operatorConfig.operatorContractAddress)
+
+            // await wait(5000) // wait for events to be processed // wait for events to be processed
+
+            logger.trace('deploying sponsorship contract')
+            const sponsorship = await deploySponsorship(config, adminWallet, {
+                streamId: streamId1 })
+            logger.trace('sponsoring sponsorship contract')
+            await (await token.connect(flagger.operatorWallet).approve(sponsorship.address, parseEther('500'))).wait()
+            await (await sponsorship.connect(flagger.operatorWallet).sponsor(parseEther('500'))).wait()
+
+            logger.trace('each operator delegates to its operactor contract')
+            logger.trace('delegating from flagger: ' + flagger.operatorWallet.address)
+            await (await token.connect(flagger.operatorWallet).transferAndCall(flagger.operatorContract.address,
+                parseEther('200'), flagger.operatorWallet.address)).wait()
+            logger.trace('delegating from target: ' + target.operatorWallet.address)
+            await (await token.connect(target.operatorWallet).transferAndCall(target.operatorContract.address,
+                parseEther('200'), target.operatorWallet.address)).wait()
+        
+            await wait(3000) // sometimes these stake fail, possibly when they end up in the same block
+            logger.trace('staking to sponsorship contract from flagger and target and voter')
+            logger.trace('staking from flagger: ' + flagger.operatorContract.address)
+            await (await flagger.operatorContract.stake(sponsorship.address, parseEther('150'))).wait()
```

Comment:
Constant here and at line 126

Reply:
again they don't need to be equal, and making a const for every number, that is then used once in the next line just makes it less readable